test/rethinkdb/joins_test.clj

```diff
@@ -0,0 +1,51 @@
+(ns rethinkdb.joins-test
+  (:require [clojure.test :refer :all]
+            [rethinkdb.query :as r]
+            [rethinkdb.test-utils :as u]))
+
+(def names [{:id 1 :name "Pikachu", :type 2},
+            {:id 2 :name "Magnemite", :type [1 2]}])
```

Comment:
Done, see https://github.com/apa512/clj-rethinkdb/pull/175/commits/eb182465091b8e7005465a7b8eeb339e36f1ab1d.

I added a TODO in the commit to remind me of adding a test that maps the joins on the the types of Magnemite.